MiniAn/minian_text_def.py

```diff
@@ -0,0 +1,35 @@
+# For minian_main
```

Comment:
Maybe minian_definitions.py

Reply:
Is it possible to define namespaces?

Reply:
No :/ 
People are using `class:`  to do that type of thing.

And do something like that

```
class Definitions:
         blabla = "blabla"
         bob = "bob"


definition = Definitions()

definition.blabla

```

I can try to do something like that.
Especially if I put dictionary keys in the `minian_definitions.py`